client/src/features/NavBar/LoggedInFeatures.js

```diff
@@ -73,12 +75,24 @@ const ItemTab = withStyles(theme => ({
 }))(props => <Tab disableRipple {...props} />);
 
 const LoggedInFeatures = (props) => {
-  const { userName } = props;
   const classes = useStyles();
   const [value, setValue] = useState(0);
   const [toggle, handleToggle] = useState(null);
+  const [ownerName, setOwnerName] = useState('')
 
   const path = window.location.pathname.toLowerCase();
+  
+  useEffect(() => {
+    getOwnerName();
+  }, [])
+
+  const getOwnerName = () => {
+    apiRequest('GET', `/ownername`)
+    .then( result => setOwnerName(result.owner_name))
+    .catch( e => {
+      console.log(e);
```

Comment:
Another way to do it is to pass the logged in user's name on the response when user logs in, and then you can pass that through the nav bar

Reply:
To reuse the owner_name, the highest component is on loggedInFeatures. Or else I need pass down the owner_name on every component page to the nav bar.